lib/jira/resource/attachment.rb

```diff
@@ -20,8 +20,10 @@ def self.meta(client)
       end
 
       def save!(attrs)
+        file = attrs['file'] || attrs[:file] # Keep supporting 'file' parameter as a string for backward compatibility
+        mime_type = attrs[:mimeType] || 'application/binary'
```

Comment:
It's common practice in ruby to use camelcase for symbols, methods and variables (https://github.com/rubocop-hq/ruby-style-guide#snake-case-symbols-methods-vars). 
Preferably stick to that.

Reply:
Thanks for the review, I totally agree with you!

I initially went with `mime_type`, but I decided to use `mimeType` instead to mimic how the other calls to the save method works in this library.

Example from the `example.rb` file:

```
link = client.Issuelink.build
link.save(
    {
        :type => {:name => 'Relates'},
        :inwardIssue => {:key => 'AL-1'},
        :outwardIssue => {:key => 'AL-2'}
    }
)
```

Additionally, the response from the JIRA REST API contains a `mimeType` value. I thought that having consistency here would be good.

Totally open to snake_casing it if you don't think these are good arguments.

Reply:
Ah, i see. It's not so much the save method that works this way (it just converts the params to json) But the Jira API that needs the camel casing (https://docs.atlassian.com/software/jira/docs/api/REST/7.12.0/#api/2/issue/{issueIdOrKey}/attachments-addAttachment) You can ignore my previous comment! 👍